Firestore/Protos/protos/firestore/local/mutation.proto

```diff
@@ -55,4 +55,16 @@ message WriteBatch {
 
   // The local time at which the write batch was initiated.
   google.protobuf.Timestamp local_write_time = 3;
+
+  // A list of "writes" that represent a partial base state from when this
```

Comment:
Using quotes like this is fairly subtle and doesn't necessarily come across to readers who don't read English fluently. Better to just spell out how these aren't real writes. Consider "synthetic writes" or "pseudo-writes" (without quotes) instead.